src/main/java/me/contaria/seedqueue/SeedQueue.java

```diff
@@ -71,6 +73,27 @@ public void onInitializeClient() {
         }
     }
 
+    private static void logSystemInformation() {
+        int availableProcessors = Runtime.getRuntime().availableProcessors();
+        int totalProcessors = getTotalProcessors();
+
+        LOGGER.info("System Information (Logged by SeedQueue):");
+        LOGGER.info("Operating System: {}", System.getProperty("os.name"));
+        LOGGER.info("OS Version: {}", System.getProperty("os.version"));
+        LOGGER.info("CPU: {}", System.getenv("PROCESSOR_IDENTIFIER"));
+        LOGGER.info("Java Version: {}", System.getProperty("java.version"));
+        LOGGER.info("JVM Arguments: {}", String.join(" ", java.lang.management.ManagementFactory.getRuntimeMXBean().getInputArguments()));
```

Comment:
No need to specify `java.lang.management.ManagementFactory`, just `ManagementFactory` works.